docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -542,6 +542,24 @@ You may have trouble in the configuration if you see Pod statuses like `RunConta
 
 [ubuntu-vagrantfile]: https://github.com/errordeveloper/k8s-playground/blob/22dd39dfc06111235620e6c4404a96ae146f26fd/Vagrantfile#L11)
 
+1. If you see error while run kubectl get po
```

Comment:
If you are seeing the following error when running for example `kubectl get po`, there
might be a certificate mismatch problem to be fixed.